apps/jobs/models.py

```diff
@@ -140,6 +140,9 @@ class Submission(TimeStampedModel):
     # Store the values of meta attributes for the submission here.
     submission_metadata = JSONField(blank=True, null=True)
     is_verified_by_host = models.BooleanField(default=False)
+    
+    #flag for submission type: UI(False) and CLI(True)
+    isSubmissionCliType = models.BooleanField(default=True)
```

Comment:
Make sure the variable name is consistent with others. In python we have vars separated by `_` instead of camel casing.

Reply:
It should be `from_cli` default to `False`.